doc/howto/json.md

```diff
@@ -0,0 +1,130 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Querying JSON files
+
+In this tutorial, we'll show you how to query JSON with JupySQL and [DuckDB](https://duckdb.org/docs/extensions/json.html).
+
+
+First, let's install the required dependencies:
+
+```{code-cell} ipython3
+%pip install jupysql duckdb duckdb-engine rich --quiet
```

Comment:
Seeing this error `ERROR: pip's dependency resolver does not currently take into account...` . Maybe output of this cell can be hidden?

Reply:
done